networks_test.go

```diff
@@ -393,6 +393,120 @@ func TestListNetworkEmpty(t *testing.T) {
 	}
 }
 
+func TestListNetworkFailure(t *testing.T) {
+	ts := newServer(response{200, `
+{"listNetworksresponse": {
+	"count": 3456,
```

Comment:
Interesting, does it paginate?

Reply:
Yes but on the real number of items and not 3456 times

Reply:
right, `if len(Network) == 0; break`

Reply:
@greut https://github.com/exoscale/egoscale/issues/81